scripts/terraform_schema.py

```diff
@@ -0,0 +1,42 @@
+"""
+Save Terraform version and schema information to the file specified
+in AZUL_TRACKED_SCHEMA_PATH or verify that the Terraform information
+in that file is up-to-date.
+"""
+import argparse
+import logging
+
+from azul import (
+    config,
+)
+from azul.deployment import (
+    terraform,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+
+log = logging.getLogger(__name__)
+
+
+def check_schema_json() -> None:
+    if terraform.versions() == terraform.tracked_schema['versions']:
```

Comment:
Still leaky. It leaks the fact that the schema is stored as a dictionary and that one of the keys in that dictionary is 'versions`.

Reply:
Hannes says versions and schema subkeys should be exposed through the Terraform class as properties, and that there should be a method that takes those two properties and writes them to a file.